src/js/containers/search/filters/budgetCategory/BudgetCategorySearchContainer.jsx

```diff
@@ -30,10 +49,16 @@ export class BudgetCategorySearchContainer extends React.Component {
 
     updateBudgetFunctions(budgetFunction) {
         this.props.updateSelectedBudgetFunctions(budgetFunction);
+
+        // Analytics
+        BudgetCategorySearchContainer.logBudgetFunctionFilterEvent(budgetFunction.title);
     }
 
     updateFederalAccounts(federalAccount) {
         this.props.updateSelectedFederalAccounts(federalAccount);
+
+        // Analytics
+        BudgetCategorySearchContainer.logFederalAccountFilterEvent(federalAccount.federal_account_code);
```

Comment:
Would we want to turn these into a single `logFilterEvent` that would take both the type of event as well as the budget function title / federal account code?